docs/getting-started-guides/juju.md

```diff
@@ -205,54 +213,49 @@ The [k8petstore example](../../examples/k8petstore/) is available as a
 
     juju action do kubernetes-master/0
 
-Note: this example includes curl statements to exercise the app, which automatically generates "petstore" transactions written to redis, and allows you to visualize the throughput in your browser.
+> Note: this example includes curl statements to exercise the app, which
+> automatically generates "petstore" transactions written to redis, and allows
+> you to visualize the throughput in your browser.
 
 ## Tear down cluster
 
     ./kube-down.sh
 
-or
+or destroy your current Juju environment (using the `juju env` command):
 
     juju destroy-environment --force `juju env`
 
+
 ## More Info
 
-Kubernetes Bundle on Github
+The Kubernetes charms and bundles can be found in the `kubernetes` project on
+github.com:
 
- - [Bundle Repository](https://github.com/whitmo/bundle-kubernetes)
-   * [Kubernetes master charm](https://github.com/whitmo/charm-kubernetes-master)
-   * [Kubernetes node charm](https://github.com/whitmo/charm-kubernetes)
- - [Bundle Documentation](http://whitmo.github.io/bundle-kubernetes)
- - [More about Juju](https://juju.ubuntu.com)
+ - [Bundle Repository](https://github.com/GoogleCloudPlatform/kubernetes/tree/master/cluster/juju/bundles)
```

Comment:
Please use http://releases.k8s.io/HEAD/..., such as http://releases.k8s.io/HEAD/cluster/juju/bundles, for the URLs. This will ensure it points to the appropriate version for each release.